classes/user/form/ResetPasswordForm.inc.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/**
+ * @file classes/user/form/ChangePasswordForm.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class ChangePasswordForm
+ * @ingroup user_form
+ *
+ * @brief Form to change a user's password.
+ */
+
+
+import('lib.pkp.classes.form.Form');
+
+class ResetPasswordForm extends Form {
+
+	/** @var object */
+	protected $_user;
+
+	/** @var object */
+	protected $_site;
+
+    /** @var string */
```

Comment:
There are some space-based indents mixed in here and elsewhere.

Reply:
Should be fixed now .

Reply:
Looks like some space-based indents got added back to ResetPasswordForm (and possibly elsewhere).

Reply:
This looks fine to me if I view from `Files Changed` section . It seems like above file link referencing the older commit ref.

Reply:
See e.g.: https://github.com/pkp/pkp-lib/pull/8450/files#diff-aedab8ceb7dcf1bc33ab9603457b7a800e5664d2efa62ee0f6aafd0fded7e3c8R126

Reply:
Should be fixed now .